<div class="splash" id="researchsplash">

	<div class="row splashyla">
        <div class="col-md-12">
            <a href="http://2014.igem.org/"><img src="http://2014.igem.org/wiki/images/0/09/Aalto_Helsinki_Logov_iGEM.png" class="img-responsive igem-logo"></a>
        </div>
    </div>

	<div class= "row splashesittely">
		<div class="col-md-3">
		</div>
		<div class="col-md-6">
			<h1>Research</h1>
			<p class="bigsplashtext">
				We engineered a three-channel switch that allows you to choose which of its three genes is active. The switch is designed so that the user can define the expressed genes independently.
			</p>
		</div>
		<div class="col-md-3">
		</div>
	</div>

	<div class= "row splashala">
        <div class ="col-md-12 transp scroller">
            <a href="#Switch">
                Scroll down to read more
                <img src="http://2014.igem.org/wiki/images/3/3e/Aalto_Helsinki_Nuoli.png" class="img-responsive center-block transp nuoli">
            </a>
        </div>
    </div>

	<div class="splashalavalikko">
		<a href="#Switch">The Switch</a><br>
        <a href="#Background">Background</a><br>
        <a href="#Hypothesis">Hypothesis</a><br>
        <a href="#Parts">Parts</a><br>
        <a href="#Methods">Methods</a><br>
        <a href="#Results">Results</a><br>
		<a href="#Discussion">Discussion</a><br>
        <a href="#Conclusions">Conclusions</a><br>
		<a href="#References">References</a><br>
    </div>

	<div class = "splashcopyright">
        <p class="transp">
            <a href="http://i-see-faces.deviantart.com/">Photo © Tanja Maria</a>
        </p>
    </div>
</div>

<script src="http://2014.igem.org/Team:Aalto-Helsinki/jquery-1.11.0.min.js?action=raw&ctype=text/js"></script>
<script src="http://2014.igem.org/Team:Aalto-Helsinki/bootstrap.min.js?action=raw&ctype=text/js"></script>
<script src="http://2014.igem.org/Team:Aalto-Helsinki/waypoints.min.js?action=raw&ctype=text/js"></script>
<div class="navbar navbar-default extrabar" data-spy="affix">
    <nav>
        <a id="Switch-submenu" class="active" href="#Switch">The Switch</a>
        <a id="Background-submenu" href="#Background">Background</a>
        <a id="Hypothesis-submenu" href="#Hypothesis">Hypothesis</a>
        <a id="Parts-submenu" href="#Parts">Parts</a>
        <a id="Methods-submenu" href="#Methods">Methods</a>
        <a id="Results-submenu" href="#Results">Results</a>
        <a id="Discussion-submenu" href="#Discussion">Discussion</a>
        <a id="Conclusions-submenu" href="#Conclusions">Conclusions</a>
        <a id="References-submenu" href="#References">References</a>
    </nav>
</div>
<div class="update Switch"></div>
<article>
	<div class="link" id="Switch"></div>
	<h2>The Three-Channel Switch</h2>

	<h3>Introduction</h3>
	<p>
		We engineered a three-channel switch that can be controlled with the intensity of blue light. By utilizing the mechanisms of lambda (λ) repressor and linking it to a blue light sensor protein, we would be able to swiftly switch between the expressions of three different genes. This mechanism could provide a nearly real-time control over the chosen genes, which could advance a variety of industrial bioprocesses, speed up research projects and benefit metabolic engineering. The switch has a modular structure, and thus, the users can decide the genes themselves without needing to modify the actual mechanism at all.
	</p>

	<div class="row margintop">
		<div class="col-md-8 text-col-left">
			<h3>Features</h3>
			<p>
				We chose YF1 fusion protein as the light receptor protein. The receptor autophosphorylates in darkness but in blue light, it is unphosphorylated. A phosphorylated YF1 protein acts as a kinase and activates FixJ transcription factor, which can then bind to a <i>FixK<sub>2</sub></i> binding site and activate the production of λ repressor protein CI. The kinase activity of YF1 is inversely proportional to blue light intensity and the effect is carried on to the acitve FixJ concentration. Thus, CI protein is not produced in bright blue light but the production increases when the blue light is dimmed down.
			</p>
			<p>
				We used lambda repressor protein CI to regulate the genes in the switch.  Only <i>gene A</i> is active when there is no repressor protein CI. At medium concentrations of CI, <i>gene A</i> becomes deactivated and <i>gene B</i> is activated. At high concentrations of CI, only <i>gene C</i> is active. The activity and deactivity of the <i>gene C</i> is based on tetracycline repressor protein that is produced with <i>genes A</i> and <i>B</i>. When neither of those genes are active, <i>gene C</i> is activated. The activities of <i>genes A</i> and <i>B</i> are controlled with an interesting mechanism of lambda repressor, which is explained thoroughly under the background title on this page.
			</p>
			<p>
				So, to put it all together, in blue light the switch activates <i>gene A</i>, in dim blue light <i>gene B</i>, and in darkness <i>gene C</i>. The image illustrates this functionality.
			</p>
		</div>
		<div class="col-md-4 img-100">
			<img src="http://2014.igem.org/wiki/images/f/fe/Aalto_Helsinki_Lamppukuva.png" class="img-responsive">
			<p class="kuvateksti">
				In blue light the switch activates <i>gene A</i>, in dim blue light <i>gene B</i>, and in darkness <i>gene C</i>. The changes are based on differences in the concentration of the λ repressor protein CI.
			</p>
		</div>
	</div>

	<h3>Glossary</h3>
    <ul>
		<li><strong>YF1 fusion protein</strong> = blue-light sensor that becomes unphosphorylated in blue light and phosphorylated (activated) in darkness</li>
		<li><strong>FixJ protein</strong> = after being phosphorylated by YF1, this activates promoter <i>FixK<sub>2</sub></i></li>
		<li><i><strong>FixK<sub>2</sub></i> binding site</strong> = activates the production of CI when FixJ binds to it</li>
		<li><strong>lambda (λ) repressor protein CI</strong> = a protein that can repress and/or activate the transcription of two different genes</li>
		<li><strong><i>O<sub>R</sub></i>  tripartite operator site</strong> = an operator site to which CI binds, downstream from <i>O<sub>L</sub></i> in λ phage</li>
		<li><strong><i>O<sub>L</sub></i> tripartite operator site</strong> = and operator site to which CI binds, upstream from <i>O<sub>R</sub></i> in λ phage</li>
		<li><strong><i>P<sub>RM</sub></i> promoter</strong> = a promoter that is active only when there’s no CI protein</li>
		<li><strong><i>P<sub>R</sub></i> promoter</strong> = a promoter that is active only when there’s little CI protein, too much or too little inhibit the production</li>
		<li><strong><i>genes A-C</i></strong> = the three genes that you could insert to our system and they would be expressed as explained here</li>
		<li><strong>Tetracycline repressor protein TetR</strong> = can bind to TetR repressible promoter sites and inhibit gene transcription</li>
    </ul>
	</p>

</article>
<div class="update Switch"></div>
<div class="update Background"></div>
<article>
	<div class="link" id="Background"></div>

	<h2>Background</h2>
	<h3>Idea development</h3>
	<p>
		/miten alotettiin tutkiminen ja miten nyt mitäkin sit keksittiin/
	</p>
    <h3>Earlier research</h3>
    <p>
        Uppsala, STJU, Tabor
    </p>
    <h3>Light receptor</h3>
	<h3>Lambda (λ) Repressor</h3>
	<p>
		Lambda repressor plays a crucial role in our switch. The repressor is part of the lambda phage and it regulates the lysogenic an lytic states of the virus. The lambda phage genetic material can be integrated in the host genome as prophage. This phase is called lysogenic cycle and during that state, the host cell can proliferate and function normally. However, in lytic cycle the phage genetic material is separate from the host genome and it is hevily expressed to produce more viruses which eventually lyse the host cell (hence the name). Lambda repressor is the region in the lambda phage that is responsible for regulating the current state in infected host. This phage has some very interesting characteristics that are described in this subsection.
    </p>
    <p>
        In the center of the lambda repressor there is a lambda repressor protein CI. This protein represses the genes in the lytic cycle and maintains the lysogenic state. Moreover, this protein is autoregulated in a very special manner. CI dimerizes and binds into its own promoter (P<sub>RM</sub>) upregulating its own production. However, in high concentrations, CI represses its production maintaining it on an appropriate level. Dimerized CI represses a lytic cycle transcription factor Cro by bindig to operator regions before P<sub>R</sub>. The lambda phage changes the state from lysogenic cycle to lytic cycle (i.e. lysogenic induction) when the host is exposed to DNA damaging factors, such as UV light. The damage activates the SOS reponse and production of RecA protein involved in repair and maintenance of DNA. RecA has protease activity and it cleaves CI resulting in drop of CI dimer concentration, derepression of Cro and lysogenic induction. Dodd et al (2001) showed that the concentration of CI affects the capability for lysogenic induction. Thus, it is important that lambda phage maintains the CI concentration in sepcific frame in order to be able to switch to lytic cycle when the opportunity arises. High concentration ledads to inability of lysogenic induction whereas low concentration results in too high induction sensitivity.
    </p>
    <p>
        The interaction of CI protein and lambda repressor promoters is well illustrated in the figure (Dodd et al. 2001). The repressor region consists of two promoters, P<sub>RM</sub> and P<sub>R</sub> separated by common operator sites (O<sub>R</sub> 1, 2 and 3). The CI protein dimers bind DNA in these three operator sites. Johnson et al. (1979) showed that CI binds cooperatively on the three operator sites. The O<sub>R1</sub> has the highest affinity for the CI dimer which recruits another dimer to O<sub>R2</sub> site. This blocks the transcription of the P<sub>R</sub> promoter and activates P<sub>RM</sub> promoter blocking Cro production and activating CI production, respectively. Bell et al. (2000) suggested that these CI dimers at the two operator sites form a tetramer. Very similar interaction is observed between two of the three O<sub>L</sub> operator sites which are located 2,4kb away from O<sub>R</sub> operator. It was early that CI also binds to O<sub>R3</sub> and O<sub>L3</sub> sites, but in substantially higher CI concentrations (Johnson et al., 1979). This leads to repression of P<sub>RM</sub> promoter. Maurer et al. (1980) showed that 50% repression of P<sub>RM</sub> requires 15 times the normal lysogenic concentration. However, the mechanism of P<sub>RM</sub> repression is not result of only CI dimer binding in the O<sub>R3</sub> site.
    </p>
    <p>
        Dodd et al. (2001) showed that two tetramers at O<sub>L</sub> and O<sub>R</sub> sites form an octamer by looping the DNA. They proposed that this DNA looping juxtaposes O<sub>R3</sub> and O<sub>L3</sub> sites and allows them to be linked by a CI tetramer. This, in turn, would silence the P<sub>RM</sub> promoter. Thus, the lambda repressor has three different states regulated by CI concentration: lytic (low CI concentration, P<sub>R</sub> is active and P<sub>RM</sub> is inactive), lysogenic (moderate CI concentration, P<sub>R</sub> is inactive and P<sub>RM</sub> is active) and repressing lysogenic (high CI concentration, P<sub>R</sub> is inactive and P<sub>RM</sub> is inactive). This is the underlying principle of our three-channel switch. In our approach, we regulate the CI concentration with blue light intensity and replace the Cro and CI with other genes of interest. This way, we are able to use the characteristics of lambda repressor to construct the mechanism behind the three-channel switch.
    </p>
		miten ajateltu muokata omaan systeemiin
	</p>
	<p>
		The Uppsala 2011 team [2] <br>
		SJTU-BioX-Shanghai<br>
		Jeff Tabor lab<br>
	</p>
	<p>[Kuva circuitista]<br>
		The switch is based on a protein called cI (http://www.uniprot.org/uniprot/P03034). This protein coding sequence is under a promoter that is activated by FixJ, a protein that is phosphorylated and therefore activated by another protein, called YF1. YF1 is phosphorylated and activated in dark. [Lisää!]
	</p>
	<p>
		We have designed the circuit by using both already existing biobricks and synthesized parts.
	</p>
	<p>
		We are currently assembling the prototype of our system.
	</p>

</article>
<div class="update Background"></div>
<div class="update Hypothesis"></div>
<article>
    <div class="link" id="Hypothesis"></div>

  <h2>Hypothesis</h2>
  <p>
      Light-to-CI, BioBrick convenience, OR=OL
  </p>

</article>
<div class="update Hypothesis"></div>
<div class="update Parts"></div>
<article>
    <div class="link" id="Parts"></div>

    <h2>Parts</h2>

    <p>
        /parts-lista-igem-template-säätö/
    </p>
    <p>
        The final construct ended up being fairly large. The gene circuit consists of two different segments: the light sensor that produces CI according to the intensity of light and the actual switch that responds to the differences in the concentration of repressor protein CI.
    </p>
    <p>
        We used fairly many already existing parts that were in the 2014 iGEM BioBrick Distribution. The parts we created ourselves are /mitä me nyt saadaan lähetettyä/. /selitystä kustakin partista ja että mitä se tekee/.
    </p>
    <p>
        Here’s a list of all the parts we used in our gene switch. They are in the same order as in our gene circuit and each of them is color coded as follows: /hienot värikoodisysteemit/. /makee luettelo niistä brickeistä/.
    </p>

    <p>
        This is the complete sequence that we put together. The color codes are the same as in the previous list. <i>Gene A</i> has 20 Xs as placeholders, <i>gene B</i> has 20 Xs and <i>gene C</i> has 20 Xs. As anyone could decide the genes themselves, the placeholders are just to show the correct place in the sequence.
    </p>
    <p>
        In addition to these parts, we also used a GFP part for testing the response times of YF1. /ja mitä muuta nyt ollaankaan testailtu/
    </p>
    <p>
        /genecircuitkuva/ http://2014.igem.org/wiki/images/f/f4/Aalto_Helsinki_Gene_Circuit.png
        [kuvateksti] This is our gene circuit. The upper part is the light sensing segment that produces the CI protein and the lower part reacts to differences in the CI concentration and switches the gene channel. The turquoise arrows are promoters, the turquoise circles are operator sites, the light blue circles are ribosome binding sites, the gray squares are expressed genes. The promoter of YF1 gene can be any constitutive promoter.
    </p>

</article>
<div class="update Parts"></div>
<div class="update Methods"></div>
<article>
	<div class="link" id="Methods"></div>
  <h2>Our Research Methods</h2>
  <h3>LED-rig for excitation</h3>
  <p>
  To be able to shine light on our bacteria we designed a new kind of device from scratch. Inspired by the rig developed by Tabor labs<sup>1</sup> we decided to build a simpler one, only with blue LED:s.<br /><br />
  Luckily we have some electronic knowledge in our team too. With Pietu designing the rig we built a foam-padded transportable rig that can be put in an incubator overnight.<br /><br />
  <h4>Microcontroller</h4>
  The core of the rig is an Arduino Nano<sup>2</sup> microcontroller. The Arduino is responsible for the logic behind our illumination patterns.
  The patterns in our experiments are fairly simple and could in theory be done without the Arduino, but we chose to use it due to its wide popularity and because we now could make advanced (and cool) animations on our rig. Also the Arduino acts as a 5v voltage regulator which provides the voltage for the second part of the rig:
  the Adafruit 16-channel 16-bit PWM controller<sup>3</sup>.<br /><br />

  <h4>LED:s</h4>
  Light Emitting Diodes(LEDs) are current controlled components, and their intensity depends on the current passing trought them. In our project we used miniature leds because of their small power and small heat generation. It is important that the LED:s wont heat up the bacteria remarkably.
  Controlling the intensity of these leds is trivial by varying the current, but this will cause the emission spectra to shift<sup>3</sup>. Hence we choose a different, and very popular, method for varying LED intensity: Pulse Width Modulation (PWM). The idea behind PWM is to turn the led on and off very in very short interval, up to frequencies of 1.6kHz. By varying the time the led is on and off we will be able to make a percieved difference in intensity. The bigger the ratio of on-time versus off-time the brighter the LED.<br /><br />
  Due to the hurry of making this rig we chose to use a pre-made Arduino "shield" for this task. We chose the a 16-channel 16-bit PWM shield from Adafruit. The 16-bits provide us much more range than the standard 8-bits of the Arduino. Additionally the Nano does not support as many channels so having 16 LEDs provides more flexibility in our measurements.<br />
  <h4>Constructing</h4>
  We wanted to be able to illuminante a bacterial culture with a single led at multiple different intensities. We choose to grow our bacteria on microtiter plates. This choice was also affected by the fact that we had an access to a fluorometer (Thermo Scientific Varioskan<sup>5</sup>) that measures fluorescence out of such microtiter plates.<br />
  We designed our LED rig so that an unmodified 96 well microtiter plate with the test cultures can be easily inerted. Our LEDs are attached to a lid that can be placed on a standard microtiter plate. 16 LEDs will illuminate the wells on columns 11,9,7,5 and rows B, D, F, H.
  <br />
  <br />
  Our LEDs' spectrum peak at the value of 470nm. This value is specified by the manufacturer but the spectra of individual leds can vary considerably. Therefore we will measure the spectra of our LED:s with a spectrophotometer to be sure of the excitation wavelength.
  <br />We have yet to invent a way to measure the LED intensity. If you have ideas, pleaser contact us!
  <br /><br />
  <ol>
  	<li> E.J. Olson, L.A. Hartsough, B.P. Landry, R. Shroff, J.J. Tabor,<br />
"Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals."<br />
Nature Methods 11(4), 449-455 (2014).</li></li>
	<li>Arduino Nano microcontroller, http://arduino.cc/en/Main/arduinoBoardNano</li>
	<li>Adafruit 16-channel PWM shield, https://learn.adafruit.com/adafruit-16-channel-pwm-slash-servo-shield?view=all</li>
	<li>S. Muthu, F. Schuurmans and M. Pashley, “Red, Green and Blue LED based white light generation: Issues and Control,” 37th Annual IEEEIAS meeting, Vol. 1, pp. 327 – 333 (2002)</li>
	<li>http://www.thermoscientific.com/content/tfs/en/product/varioskan-flash-multimode-reader.html</li>
  </ol>
  </p>
fl
</article>
<div class="update Methods"></div>
<div class="update Results"></div>
<article>
	<div class="link" id="Results"></div>

  <h2>Results</h2>

</article>
<div class="update Results"></div>
<div class="update Discussion"></div>
<article>
	<div class="link" id="Discussion"></div>

  <h2>Discussion</h2>
  <p>
        Mutaatiot operaattorialueisiin
        Plasmidissa ei DNA:ssa
  </p>

</article>
<div class="update Discussion"></div>
<div class="update Conclusions"></div>
<article>
    <div class="link" id="Conclusions"></div>

  <h2>Conclusions</h2>

</article>
<div class="update Conclusions"></div>
<div class="update References"></div>
<article>
	<div class="link" id="References"></div>

  <h2>References</h2>

    <ol>
        <li>Peking 2012 iGEM Team: http://2012.igem.org/Team:Peking</li>
        <li>Uppsala-Sweden 2011 iGEM Team: http://2011.igem.org/Team:Uppsala-Sweden </li>
        <li>SJTU BioX Shanghai 2013 iGEM Team: http://2013.igem.org/Team:SJTU-BioX-Shanghai</li>
    </ol>

</article>
<div class="update References"></div>
