<div class="splash" id="researchsplash">

	<div class="row splashyla">
        <div class="col-md-12">
            <a href="http://2014.igem.org/"><img src="http://2014.igem.org/wiki/images/0/09/Aalto_Helsinki_Logov_iGEM.png" class="img-responsive igem-logo"></a>
        </div>
    </div>

	<div class= "row splashesittely">
		<div class="col-md-3">
		</div>
		<div class="col-md-6">
			<h1>Research</h1>
			<p class="bigsplashtext">
				We engineered a three-channel switch that allows you to choose which of its three genes is active. The switch is designed so that the user can define the expressed genes independently.
			</p>
		</div>
		<div class="col-md-3">
		</div>
	</div>

	<div class= "row splashala">
        <div class ="col-md-12 transp scroller">
            <a href="#Switch">
                Scroll down to read more
                <img src="http://2014.igem.org/wiki/images/3/3e/Aalto_Helsinki_Nuoli.png" class="img-responsive center-block transp nuoli">
            </a>
        </div>
    </div>

	<div class="splashalavalikko">
		<a href="#Switch">The Switch</a><br>
        <a href="#Background">Background</a><br>
        <a href="#Hypothesis">Hypothesis</a><br>
        <a href="#Parts">Parts</a><br>
        <a href="#Methods">Methods</a><br>
        <a href="#Results">Results</a><br>
		<a href="#Discussion">Discussion</a><br>
        <a href="#Conclusions">Conclusions</a><br>
		<a href="#References">References</a><br>
    </div>

	<div class = "splashcopyright">
        <p class="transp">
            <a href="http://i-see-faces.deviantart.com/">Photo © Tanja Maria</a>
        </p>
    </div>
</div>

<script src="http://2014.igem.org/Team:Aalto-Helsinki/jquery-1.11.0.min.js?action=raw&ctype=text/js"></script>
<script src="http://2014.igem.org/Team:Aalto-Helsinki/bootstrap.min.js?action=raw&ctype=text/js"></script>
<script src="http://2014.igem.org/Team:Aalto-Helsinki/waypoints.min.js?action=raw&ctype=text/js"></script>
<div class="navbar navbar-default extrabar" data-spy="affix">
    <nav>
        <a id="Switch-submenu" class="active" href="#Switch">The Switch</a>
        <a id="Background-submenu" href="#Background">Background</a>
        <a id="Hypothesis-submenu" href="#Hypothesis">Hypothesis</a>
        <a id="Parts-submenu" href="#Parts">Parts</a>
        <a id="Methods-submenu" href="#Methods">Methods</a>
        <a id="Results-submenu" href="#Results">Results</a>
        <a id="Discussion-submenu" href="#Discussion">Discussion</a>
        <a id="Conclusions-submenu" href="#Conclusions">Conclusions</a>
        <a id="References-submenu" href="#References">References</a>
    </nav>
</div>
<div class="update Switch"></div>
<article>
	<div class="link" id="Switch"></div>
	<h2>The Three-Channel Switch</h2>

	<h3>Introduction</h3>
	<p>
		We engineered a three-channel switch that can be controlled with the intensity of blue light. By utilizing the mechanisms of lambda (λ) repressor and linking it to a blue light sensor protein, we would be able to swiftly switch between the expressions of three different genes. This mechanism could provide a nearly real-time control over the chosen genes, which could advance a variety of industrial bioprocesses, speed up research projects and benefit metabolic engineering. The switch has a modular structure, and thus, the users can decide the genes themselves without needing to modify the actual mechanism at all.
	</p>
    <p>
        Previously gene regulation was controlled by chemical effectors, such as tetracycline and IPTG. In many cultures, a constat and uniform level of concentration is hard to maintain as the chemicals get absorbed, degraded and modified by the cells and environment. Moreover, export of these chemicals is hard and time consuming or even impossible and the response time is limited by the rate of chemical addition and diffusion. On top of all, the effector chemicals may even be toxic or bind unknown receptors leading to unwanted effects in the culture. (Schmidl et al., 2014)
    </p>
    <p>
        Light signals, on the other hand, are non-toxic and we have full control over it: it is very easy to increase and decrease the light intensity. In many cases, light is orthogonal signal that is does not affect to the indigenous cell mechanisms and pathways. Such is the case in for ecample <em>E. coli</em>. We are also able to target light very specifically to single colonies or even single cells allowing unprecedented accuracy. This allows very precise spatial and temporal control over the light signal. And lastly, the light signals can be emitted using LEDs which are widely available, durable and very inexpensive even in a large scale.
    </p>

	<div class="row margintop">
		<div class="col-md-8 text-col-left">
			<h3>Features</h3>
			<p>
				We chose YF1 fusion protein as the light receptor protein. The receptor autophosphorylates in darkness but in blue light, it is unphosphorylated. A phosphorylated YF1 protein acts as a kinase and activates FixJ transcription factor, which can then bind to a <i>FixK<sub>2</sub></i> binding site and activate the production of λ repressor protein CI. The kinase activity of YF1 is inversely proportional to blue light intensity and the effect is carried on to the acitve FixJ concentration. Thus, CI protein is not produced in bright blue light but the production increases when the blue light is dimmed down.
			</p>
			<p>
				We used lambda repressor protein CI to regulate the genes in the switch.  Only <i>gene A</i> is active when there is no repressor protein CI. At medium concentrations of CI, <i>gene A</i> becomes deactivated and <i>gene B</i> is activated. At high concentrations of CI, only <i>gene C</i> is active. The activity and deactivity of the <i>gene C</i> is based on tetracycline repressor protein that is produced with <i>genes A</i> and <i>B</i>. When neither of those genes are active, <i>gene C</i> is activated. The activities of <i>genes A</i> and <i>B</i> are controlled with an interesting mechanism of lambda repressor, which is explained thoroughly under the background title on this page.
			</p>
			<p>
				So, to put it all together, in blue light the switch activates <i>gene A</i>, in dim blue light <i>gene B</i>, and in darkness <i>gene C</i>. The image illustrates this functionality.
			</p>
		</div>
		<div class="col-md-4 img-100">
			<img src="http://2014.igem.org/wiki/images/f/fe/Aalto_Helsinki_Lamppukuva.png" class="img-responsive">
			<p class="kuvateksti">
				In blue light the switch activates <i>gene A</i>, in dim blue light <i>gene B</i>, and in darkness <i>gene C</i>. The changes are based on differences in the concentration of the λ repressor protein CI.
			</p>
		</div>
	</div>

	<h3>Glossary</h3>
    <ul>
		<li><strong>YF1 fusion protein</strong> = blue-light sensor that becomes unphosphorylated in blue light and phosphorylated (activated) in darkness</li>
		<li><strong>FixJ protein</strong> = after being phosphorylated by YF1, this activates promoter <i>FixK<sub>2</sub></i></li>
		<li><strong><em>FixK</em><sub>2</sub> binding site</strong> = activates the production of CI when FixJ binds to it</li>
		<li><strong>lambda (λ) repressor protein CI</strong> = a protein that can repress and/or activate the transcription of two different genes</li>
		<li><strong><em>O</em> <sub>R</sub> tripartite operator site</strong> = an operator site to which CI binds, downstream from <em>O</em><sub>L</sub> in λ phage</li>
		<li><strong><em>O</em><sub>L</sub> tripartite operator site</strong> = and operator site to which CI binds, upstream from <em>O</em><sub>R</sub> in λ phage</li>
		<li><strong><em>P</em><sub>RM</sub> promoter</strong> = a promoter that is active only when there’s no CI protein</li>
		<li><strong><em>P</em><sub>R</sub> promoter</strong> = a promoter that is active only when there’s little CI protein, too much or too little inhibit the production</li>
		<li><strong><em>genes A-C</em></strong> = the three genes that you could insert to our system and they would be expressed as explained here</li>
		<li><strong>Tetracycline repressor protein TetR</strong> = can bind to TetR repressible promoter sites and inhibit gene transcription</li>
    </ul>
	</p>

</article>
<div class="update Switch"></div>
<div class="update Background"></div>
<article>
	<div class="link" id="Background"></div>

	<h2>Background</h2>
	<h3>Developing Our Idea</h3>
    <div class="row">
		<div class="col-md-8 text-col-left">
			<p>
        		In the beginning of our project, from March 2013, we focused solely on coming up with new ideas for the project, building the public team website and applying for Summer of Startups 2014 and foundations with deadlines in spring. In our team meetings, we had different types of brainstorming sessions where all of us could present even the craziest ideas. We founded a shared folder where anybody could write down new ideas. We came up with disco bacteria that glow in sync with music, new kinds of bacterial drinks that change flavour with storing temperature, remote controllable bacteria, oil-eating bacteria, tooth paste bacteria that keeps up the mouth hygiene despite forgetting to brush the teeth, etc. On a Friday evening we had a vote and everybody gave a score from 1 to 5 for each idea. We made an idea gradient where the ideas were ranked according to the score from bottom left to top right.
    		</p>
		</div>
		<div class="col-md-4 img-100">
			<img src="http://2014.igem.org/wiki/images/5/5e/Aalto_Helsinki_Ideagradientti.jpg" class="img-responsive">
			<p class="kuvateksti">
				This is the amazing post-it idea gradient that we made.
			</p>
		</div>
	</div>
    <p>
        We chose the top 5 ideas and presented them to our advisors and researchers from Aalto University, VTT and University of Helsinki to get feedback and comment on the feasibility. After all, none of us had never done anything even close to this before. As a result of this feedback, further idea development and admission to Summer of Startups, we decided to work on the idea with a project name "nest box." It's a container resembling a traditional Finnish nest box but it includes bacteria producing different fragrances. These bacteria could be controlled using light concealed inside the box. The user can decide which fragrance to produce by a press of a button and this action would be translated into light signal to control the gene expression in the bacteria. Such a system is concrete and understandable for the public audience and it would help to build a positive image of synthetic biology in Finland. In theory, such system could be used in conference rooms, hotels, living rooms, saunas, movie theaters... The scent marketing is an expanding business and our idea had the potential to add something new to it. Furthermore, it was a concrete product which made our participation in Summer of Startups a lot easier.
	</p>
    <p>
        We started our BioBrick learning practice project mid May and Summer of Startups early June. As we got forward, we realized that our idea is not very realistic. Implementation of such a separate unmaintained container would be very challenging and in addition, we cannot bring genetically modified bacteria outside laboratory conditions (it's defined in the gene technology legislation). We decided to solely focus on one element of our nest box idea. Constructing the interface between the bacteria and the user or a computer is not trivial. So, we redefined our direction which was maintained to the end. We wanted to develop a three channel switch that would allow us to activate one of the three genes at a time using only the intensity of single wavelength light. We could then connect this switch with fragrance producing genes, for example.
    </p>
    <h3>Earlier Research</h3>
    <p>
        Optical control over molecular interactions has been harnessed in metabolic signaling, dimerization systems and light activated adrenergic receptors and neuronal ion channels. Most importantly in terms of synthetic biology we can now regulate transcription with light. One of the foundational papers were written in 2005 by Levskaya et al. <em>Synthetic biology: engineering Escherichia coli to see light</em>. Since then, many different light sensors and light regulators have been built. This toolbox enables more precise spatial and temporal control over the synthetic biological systems. Hence, light regulation has been part of many iGEM projects. Light sensors are arguably one of the most important elements in controlling gene expression in synthetic biology. In contrast to chemicals, we have full control over light intensity.
    </p>
    <p>
        The applications in previous iGEM teams include communication, kill switches and metabolic engineering, among many others. Previous teams have been very active in making BioBricks out of new light sensors published in research. Uppsala 2011 team designed a multichromatic light sensor system which regulates three independent genes. In their project the used blue, green and red light sensors and linked them to different chromoproteins. This way, different light signals produce different colors in bacteria. They added many useful BioBricks to the registry which has made also our project more feasible.
    </p>
    <p>
        STJU BioX Shanghai 2013 team aimed to create a universal system to regulate genomic genes using blue, red and green light and CRISPR and <em>dCas9</em> protein. Their fundamental idea is to regulate production of sgRNA with different wavelengths. This would enable regulation of three different genomic genes within the chromosome presumably independently of each other. Switching the regulatory targets would be done by changing the sgRNAs. The Shanghai team also built a box with computer controlled LEDs of all three different color. In addition, they shared the software as well as the instructions how to build such a light regulation box. This was very interesting project as we built a light regulation system for three genes using LEDs and a microcontroller. STJU BioX Shanghai gave some inspiration to our LED control system.
    </p>
    <p>
        Interestingly, also Freiburg 2013 team made a universal regulatory system using CRISPR and dCas9 protein but they targeted mammalian cells. In their concept, <em>dCas9</em> is guided to the site and the function to be carried out can be chosen remotely using different wavelengths of light. The effector either activates or represses the target gene or methylates a histone tail affecting the histone code at the site. The three effectors: repressor (KRAB), activator (VP16) and methyltransferase (G9a), can be controlled with red, blue and UV light. Such system would be able to regulate the target genes remotely in diverse ways.
    </p>
    <p>
        Maybe the most interesting research group in this field is Jeffey Tabor's group. On their website they write: "We program living cells to sense, compute, and respond to information in their environments. Our programmable cells have applications in basic science, industry, and medicine." (http://www.taborlab.rice.edu/home, 5.20.2014). They have been involved in developing new light sensors, including CcaS/CcaR and Cph8. They also constructed a multichromatic gene regulation system combining both these receptors. In a very recent paper (Schmidl et al., 2014) they re-engineered Cph8 and CcaS/CcaR sensors to reduce cross-talk and leakiness and to increase the dynamic range.
    </p>
    <p>
        Tähän ehkä vielä juttua Taborin rigistä (Olson et al., 2014)? <strong>Pietu?</strong>
    </p>
    <p>
        In contrast to the previous research, our system attempts to regulate multiple targets with only one wavelenght of light. In the paper by Tabor et al. (2011) they say: "The development of multichromatic gene regulatory systems, where different light wavelengths regulate the expression of different genes, will allow more advanced control of synthetic and natural gene regulatory networks." Whereas we completely agree with that, we also claim if we're able to use single wavelengths more efficiently, we could achieve even greater level of control of the synthetic gene regulatory networks.
    </p>
    <h3>In Search of the Perfect Light Sensing System</h3>
    <p>
        First of all, to be able to control gene expression with light, we needed a light receptor protein. It would react to certain wavelength of light and mediate the signal to the DNA level and initiate the response. We found different alternatives in the iGEM registry but most of them were not properly characterized. We could not find reliable experiences of the use of such light receptors or, when we found them, the receptor protein was not available. Our requirements for the light receptor system were following: 1) it has worked for other iGEM teams 2) it should work in <i>E. coli</i> 3) it is available in the registry 4) it does not require any cofactors or special strains 5) short response time and sensitivity. In this section, we discuss about the different alternatives we found and what made us choose the YF1 protein. We also introduce the mechanism how the light signal is conveyed in our system.
    </p>
    <p>
        Phytochrome interacting factors (PIF's) were used by some previous iGEM teams, including Freiburg 2013 and TU Munich 2012. The actual red light sensing protein is called Phytochrome B and in the active form it binds to the PIF (<a href="http://parts.igem.org/Part:BBa_K1150005" >BBa_K1150005 or <a href="http://parts.igem.org/Part:BBa_K365000" >BBa_K365000</a>). Thus, the cell also needs to express Phytochrome B. However, these PIFs were derived from <i>Arabidopsis thaliana</i>, which is an eukaryotic organism. Moreover, the response mechanism would require fusing the two proteins to the actual DNA binding domain and transcription factor. Unfortunately, Freiburg team only succeeded in 1.3 fold induction of gene expression after 48 hours of incubation in red light and TU Munich did not manage to implement the light sensing part in their kill switch.
    </p>
    <p>
        The Peking 2012 team designed an ultra-sensitive light sensor for luminescence. They combined a LOV domain, a dimerizing domain and a DNA-binding LexA domain. Such protein becomes active when the cells are illuminated with 440 - 480 nm light. In active form, the dimerizing domains facilitate formation of LexA dimer which is able to bind DNA and repress transcription in only such form. Whereas their light sensing system seems to work very well there were three issues with it. Firstly, their light sensing system was not available and secondly, it might have been too sensitive; controlling the light at very low intensities could be rather challenging. Furthermore, LexA is a part of SOS system already part of <i>E. coli</i> so it might interfere with indigenous <i>E. coli</i> genes. However, according to Peking 2012, mutated version of LexA could solve this problem.
    </p>
    <p>
        Cph8 <a href="http://parts.igem.org/Part:BBa_I15010">(BBa_I15010)</a> is a fusion of Cph1 red photoreceptor domain (660nm) and EnvZ histidine kinase domain. In absence of red light, EnvZ is active and it phosphorylates endogenous OmpR which in trun activates OmpC promoter <a href="http://parts.igem.org/Part:BBa_R0082">(BBa_R0082)</a>. Red light activates Cph1 which inhbits EnvZ kinase domain function. However, as both EnvZ, OmpR and OmpC are endogenous in <i>E. coli</i>, reliable use would require EnvZ deficient strain. The activation Cph1 requires Phycocyanobilin which can be produced with heme oxygenase <a href="http://parts.igem.org/Part:BBa_I15008">(BBa_I15008)</a> and ferrodoxin oxidoreductase <a href="http://parts.igem.org/Part:BBa_I15009">(BBa_I15009)</a>. The Cph8 was decribed by Levskaya et al. (2005) and they achieved nearly 10-fold induction. However, as EnvZ and OmpR are involved in indigenous osmolarity response, phosphorylated OmpR might also interfere with promoters encoding membrane porins.
    </p>
    <p>
        CcaS (<a href="http://parts.igem.org/Part:BBa_K592001">BBa_K592001</a>) is a cyanobacteriochrome that gets activated by autophosphorylation. This is upregulated by green light (535nm). The activated CcaS phosphorylates its response regulator CCaR (<a href="http://parts.igem.org/Part:BBa_K592002"> BBa_K592002</a>) which then induces the promoter <em>P</em><sub>cpcG2</sub> (Hirose et al., 2008; Tabor et al., 2011). As Cph8 system, this light receptor also requires Phycocyanobilin to function. In addition, CcaS is inactivated in red light (672nm) allowing higher degree of control and fast response times. According to the experiments by Tabor et al. (2011), the promoter <em>P</em><sub>cpcG2</sub> showed only 2-fold induction in green light. The transcription response is somewhat linear to the light intensity of 0.01 W/m<sup>2</sup> after which is rapidly saturates. Although CcaR and CcaS BioBricks are indeed available although the experiences of other iGEM teams were very few and poorly described.
    </p>

    <h3>YF1 Light Sensor</h3>
    <p>
        YF1 (<a href="http://parts.igem.org/Part:BBa_K592004">BBa_K592004</a>) is an engineered blue light (previous experiments in 430nm) sensor protein that consists of two initially separate domains, LOV domain (sensory domain) from YtvA and histidine kinase domain (effector domain) from FixL. This fusionprotein was designed and constructed by Möglich et al. (2009). YF1, nor it's precursors, do not exist naturally in <i>E. coli</i> nor reportedly interfere with indigenous <i>E. coli</i> genes but according to Möglich et al. (2009) the sensor works well in <i>E. coli</i>: it's capable of 70-fold supression of the target promoter when illuminated with blue light. The sensor does not require additional, foreign molecules in order to get activated by blue light. The gene is only 1132 bp long and it has been used by previous iGEM teams, including Uppsala 2011, STJU BioX Shanghai and Tokyo NoKoGen. Thus, we evaluated YF1 to be the most suitable for our three channel switch.
    </p>
    <p>
        Based on the work of Möglich et al. (2009) we describe the mechanism of YF1 blue light sensor in greater detail. YF1 is a water soluble chimeric fusionprotein that contains two domains linked together with a Jα linker sequence. The N terminal domain is a flavin nucleotide binding LOV domain (light-oxygen-voltage, a sub group of PAS sensor domains) from <i>Bacillus subtilis</i> YtvA. This domain incorporates noncovalently flavin mononucleotide and blue light facilitates formation of a covalent bond with a cysteine residue of the LOV domain. This induces a conformational change in this light sensory domain.
    </p>
    <p>
        The C terminal domain of YF1 is a histidine kinase, the FixL effector domain from <i>Bradyrhizobium japonicum</i>. The original protein, FixL, is involved in regulating the nitrate respiration and nitrate fixation. The histidine kinase functions only when it is phosphorylated. It has autophosphorylation activity that is regulated by the sensor domain. Autophosphorylated histidine kinase activates a FixJ (<a href="http://parts.igem.org/Part:BBa_K592005">BBa_K592005</a>) response regulator by transferring a phosphoryl group. However, when the histidine kinase domain is unphosphorylated, it has actually phosphatase activity for the response regulator. Phosphorylated FixJ binds to <em>FixK</em><sub>2</sub> promoter (<a href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</a>) initiating the expression of the corresponding genes. This system should be completely orthogonal in <i>E. coli</i>.
    </p>
    <p>
        Möglich et al. (2009) proposed that the YF1 proteins form dimers. This would also bring the histdine kinase domains close to each other. It is proposed that the autophosphorylation actually work in trans within the dimer. They hypothesized the Jα helix linker to form a coiled coil with another YF1 protein. The linker would mediate the signal from the sensor domain to the effector domain inhibiting/enabling its kinase activity. The conformational change from the incorporated flavin domain is suggested to rotate the linker helix. The relative angle of the two domains greatly affect to the mechanism. If we add one amino acid to the Jα linker, the relative angle changes and the effect of the sensor domain is inverted: in presence of light the histidine kinase is active but in darkness, it is inactive. This presents interesting approaches and easy solution to invert the YF1 signal.
    </p>
        In our three channel switch, we fused a CI protein to the <i>Bradyrhizobium japonicum</i> <em>FixK</em><sub>2</sub> promoter. Thus, in the absence of blue light (470nm in our experiments), the angle between YF1 domains autophosphorylation of histidine kinase of YF1, which then transfers a phosphoryl group to FixJ activating it. The P-FixJ in binds to <em>FixK</em><sub>2</sub> promoter activating the CI transcription. The role of this CI protein in our switch is described in the next section.
    </p>

	<h3>Lambda (λ) Repressor</h3>
	<p>
		Lambda repressor plays a crucial role in our switch. The repressor is part of the lambda phage and it regulates the lysogenic an lytic states of the virus. The lambda phage genetic material can be integrated in the host genome as prophage. This phase is called lysogenic cycle and during that state, the host cell can proliferate and function normally. However, in lytic cycle the phage genetic material is separate from the host genome and it is hevily expressed to produce more viruses which eventually lyse the host cell (hence the name). Lambda repressor is the region in the lambda phage that is responsible for regulating the current state in infected host. This phage has some very interesting characteristics that are described in this subsection.
    </p>
    <p>
        In the center of the lambda repressor there is a lambda repressor protein CI (<a href="http://parts.igem.org/Part:BBa_K105004">BBa_K105004</a>). This protein represses the genes in the lytic cycle and maintains the lysogenic state. Moreover, this protein is autoregulated in a very special manner. CI dimerizes and binds into its own promoter (<em>P</em><sub>RM</sub>) upregulating its own production. However, in high concentrations, CI represses its production maintaining it on an appropriate level. Dimerized CI represses a lytic cycle transcription factor Cro by bindig to operator regions before <em>P</em><sub>R</sub>. The lambda phage changes the state from lysogenic cycle to lytic cycle (i.e. lysogenic induction) when the host is exposed to DNA damaging factors, such as UV light. The damage activates the SOS reponse and production of RecA protein involved in repair and maintenance of DNA. RecA has protease activity and it cleaves CI resulting in drop of CI dimer concentration, derepression of Cro and lysogenic induction. Dodd et al (2001) showed that the concentration of CI affects the capability for lysogenic induction. Thus, it is important that lambda phage maintains the CI concentration in sepcific frame in order to be able to switch to lytic cycle when the opportunity arises. High concentration ledads to inability of lysogenic induction whereas low concentration results in too high induction sensitivity.
    </p>
    <p>
        The interaction of CI protein and lambda repressor promoters is well illustrated in the figure (Dodd et al. 2001). The repressor region consists of two promoters, <em>P</em><sub>RM</sub> and <em>P</em><sub>R</sub> separated by common operator sites (<em>O</em><sub>R</sub> 1, 2 and 3). The CI protein dimers bind DNA in these three operator sites. Johnson et al. (1979) showed that CI binds cooperatively on the three operator sites. The <em>O</em><sub>R</sub>1 has the highest affinity for the CI dimer which recruits another dimer to <em>O</em><sub>R</sub>2 site. This blocks the transcription of the <em>P</em><sub>R</sub> promoter and activates <em>P</em><sub>RM</sub> promoter blocking Cro production and activating CI production, respectively. Bell et al. (2000) suggested that these CI dimers at the two operator sites form a tetramer. Very similar interaction is observed between two of the three <em>O</em><sub>L</sub> operator sites which are located 2,4kb away from <em>O</em><sub>R</sub> operator. It was early that CI also binds to <em>O</em><sub>R</sub>3 and <em>O</em><sub>L</sub>3 sites, but in substantially higher CI concentrations (Johnson et al., 1979). This leads to repression of <em>P</em><sub>RM</sub> promoter. Maurer et al. (1980) showed that 50% repression of <em>P</em><sub>RM</sub> requires 15 times the normal lysogenic concentration. However, the mechanism of <em>P</em><sub>RM</sub> repression is not result of only CI dimer binding in the <em>O</em><sub>R</sub>3 site.
    </p>
    <p>
        Dodd et al. (2001) showed that two tetramers at <em>O</em><sub>L</sub> and <em>O</em><sub>R</sub> sites form an octamer by looping the DNA. They proposed that this DNA looping juxtaposes <em>O</em><sub>R</sub>3 and <em>O</em><sub>L</sub>3 sites and allows them to be linked by a CI tetramer. This, in turn, would silence the <em>P</em><sub>RM</sub> promoter. Thus, the lambda repressor has three different states regulated by CI concentration: lytic (low CI concentration, <em>P</em><sub>R</sub> is active and <em>P</em><sub>RM</sub> is inactive), lysogenic (moderate CI concentration, <em>P</em><sub>R</sub> is inactive and <em>P</em><sub>RM</sub> is active) and repressing lysogenic (high CI concentration, <em>P</em><sub>R</sub> is inactive and <em>P</em><sub>RM</sub> is inactive). This is the underlying principle of our three-channel switch. In our approach, we regulate the CI concentration with blue light intensity and replace the Cro and CI with other genes of interest. This way, we are able to use the characteristics of lambda repressor to construct the mechanism behind the three-channel switch.
    </p>

</article>
<div class="update Background"></div>
<div class="update Hypothesis"></div>
<article>
    <div class="link" id="Hypothesis"></div>

  	<h2>Hypotheses</h2>
  	<p>
  		The very first and critical assumption is that the concentration of CI would be controlled by light when it is put behind <em>FixK</em><sub>2</sub> promoter. Moreover, we assume that this effect will be more or less linear on the relevant region so that double light intensity leads eventually to double concentration. It is also very crucial that <em>FixK</em><sub>2</sub> promoter does not show significant leaky expression in absence of phosphorylated FixJ. We need to be able to keep CI concentration low enough to keep <em>P</em><sub>R</sub> promoter active in presence of strong blue light.
    </p>
    <p>
        To better match the BioBrick standards and the idea about being able to ligate BioBricks together as is, we wanted to make our promoters unidirectional. The <em>P</em><sub>RM</sub> and <em>P</em><sub>R</sub> promoters, however, point to opposite directions. This would require flipping the BioBricks added to the preceding strand or using reverse complement BioBricks. This would not be very convenient. Hence, we make a duplication of the OR sites, one coupled with <em>P</em><sub>RM</sub> and the other connected to <em>P</em><sub>R</sub> promoter. We suggest we are able to repress <em>P</em><sub>R</sub> promoter and activate <em>P</em><sub>R</sub> promoter with CI dimers despite this duplication.
    </p>
    <p>
        The third hypothesis is related to the DNA looping as a result of CI octamerization at the operator sites. Originally, DNA looping takes place between <em>O</em><sub>R</sub> and <em>O</em><sub>L</sub> sites by octamerization of CI dimers bound to operator sites 1 and 2 ab both locations. As <em>O</em><sub>R</sub> and <em>O</em><sub>L</sub> regions resemble each other in terms of CI affinity and as the interacting factor seems to be CI and not the DNA sequence, we hypothesize that <em>O</em><sub>L</sub> could be replaced by <em>O</em><sub>R</sub> region without losing the DNA looping capability. This would also underline the significance of the previous assumption: as we have duplicated the <em>O</em><sub>R</sub> sites we already have the hypotetical factors required for DNA looping and <em>P</em><sub>RM</sub> repression. Therefore, if we duplicate the <em>O</em><sub>R</sub> region and separate them with correct number of bases, we might be able to form the DNA loop between these regions without having a <em>O</em><sub>L</sub> region.
    </p>
    <p>
        The lambda repressor functions in lambda profage when the virus is in the lysogenic state. This means the lambda repressor is integrated as a part of the genome. It has been shown that the DNA looping takes place in the infected <i>E. coli</i> cells. However, we have not found evidence that this would happen also in the plasmids containing the lambda repressor region. It is possible the supercoiled structure of the plasmid prevents the normal function of the lambda repressor. We have identified three possible scenarios. Firstly, the lambda repressor could function within the plasmid as is. Alternatively, the loop size formed by lambda repressor may be changed or it may be easier or more difficult to be formed but it may work with some modifications. The worst scenario is that the lambda repressor does not work at all in the plasmid context. In such case, our three channel switch could be integrater to the <em>E. coli</em> genome.
  	</p>
  	<p>
  		Lastly, we made some hypotheses regarding CI concentration. The wild type CI half life has been measured to be longer than 10 hours (Parsell et al., 1990). This would mean that it takes very long time to reduce the CI concentration that is, switch back from the channel C in our system. However, we can fuse an ssrA tag to the protein an target it for faster degradation within the cell. According to Keiler et al. (1996), such tagging can reduce the half life to 4 minutes. Hence, we used LVA tagged version of CI lambda repressor protein (<a href="http://parts.igem.org/Part:BBa_K327018">BBa_K327018</a>). This enables much shorter reaction time and nearly real-time control of CI concentration. However, it also lowers the highest possible concentration of CI. We hypothesize, that using strong enough promoter, we can reach sufficient CI concentrations for <em>P</em><sub>RM</sub> repression.
  	</p>

</article>
<div class="update Hypothesis"></div>
<div class="update Parts"></div>
<article>
    <div class="link" id="Parts"></div>

    <h2>Parts</h2>

    <p>
        /parts-lista-igem-template-säätö/
    </p>
    <p>
        The final construct ended up being fairly large. The gene circuit consists of two different segments: the light sensor that produces CI according to the intensity of light and the actual switch that responds to the differences in the concentration of repressor protein CI.
    </p>
    <p>
        We used fairly many already existing parts that were in the 2014 iGEM BioBrick Distribution. The parts we created ourselves are /mitä me nyt saadaan lähetettyä/. /selitystä kustakin partista ja että mitä se tekee/.
    </p>
    <p>
        Here’s a list of all the parts we used in our gene switch. They are in the same order as in our gene circuit and each of them is color coded as follows: /hienot värikoodisysteemit/. /makee luettelo niistä brickeistä/.
    </p>

    <p>
        This is the complete sequence that we put together. The color codes are the same as in the previous list. <i>Gene A</i> has 20 Xs as placeholders, <i>gene B</i> has 20 Xs and <i>gene C</i> has 20 Xs. As anyone could decide the genes themselves, the placeholders are just to show the correct place in the sequence.
    </p>
    <p>
        In addition to these parts, we also used a GFP part for testing the response times of YF1. /ja mitä muuta nyt ollaankaan testailtu/
    </p>

    <div class="row">
		<div class="img-center">
			<img src="http://2014.igem.org/wiki/images/f/f4/Aalto_Helsinki_Gene_Circuit.png" class="img-responsive"></img>
			<p class="kuvateksti">
				This is our gene circuit. The upper part is the light sensing segment that produces the CI protein and the lower part reacts to differences in the CI concentration and switches the gene channel. The turquoise arrows are promoters, the turquoise circles are operator sites, the light blue circles are ribosome binding sites, the gray squares are expressed genes. The promoter of YF1 gene can be any constitutive promoter.
			</p>
		</div>
	</div>

</article>
<div class="update Parts"></div>
<div class="update Methods"></div>
<article>
	<div class="link" id="Methods"></div>
  	<h2>Our Research Methods</h2>

  	<h3>LED Rig</h3>
  	<div class="row">
		<div class="col-md-6 text-col-left">
			<p>
  				To be able to shine light on our bacteria we designed a new kind of device from scratch. Inspired by the rig developed by Tabor labs<sup>1</sup> we decided to build a simpler one, only with blue LED:s.
  			</p>
  			<p>
  				Luckily we had some electronic knowledge in our team, too. With Pietu designing the rig we built a foam-padded transportable rig that can be put in an incubator overnight.
  			</p>
		</div>
		<div class="col-md-6 img-100">
			<img src="http://2014.igem.org/wiki/images/4/43/Aalto_Helsinki_Peityled.jpg" class="img-responsive">
			<p class="kuvateksti">
				Pietu presents the great LED rig.
			</p>
		</div>
	</div>

  	<h4>Microcontroller</h4>
  	<p>
  		The core of the rig is an Arduino Nano<sup>2</sup> microcontroller. The Arduino is responsible for the logic behind our illumination patterns.
  	</p>
  	<p>
  		The patterns in our experiments are fairly simple and could in theory be done without the Arduino, but we chose to use it due to its wide popularity and because we now could make advanced (and cool) animations on our rig. Also the Arduino acts as a 5v voltage regulator which provides the voltage for the second part of the rig: the Adafruit 16-channel 16-bit PWM controller<sup>3</sup>.
  	</p>

  	<h4>LEDs</h4>
  	<p>
  		Light Emitting Diodes(LEDs) are current controlled components, and their intensity depends on the current passing trought them. In our project we used miniature leds because of their small power and small heat generation. It is important that the LED:s wont heat up the bacteria remarkably.
  	</p>
  	<p>
  		Controlling the intensity of these leds is trivial by varying the current, but this will cause the emission spectra to shift<sup>3</sup>. Hence we choose a different, and very popular, method for varying LED intensity: Pulse Width Modulation (PWM). The idea behind PWM is to turn the led on and off very in very short interval, up to frequencies of 1.6kHz. By varying the time the led is on and off we will be able to make a percieved difference in intensity. The bigger the ratio of on-time versus off-time the brighter the LED.
  	</p>
  	<p>
  		Due to the hurry of making this rig we chose to use a pre-made Arduino "shield" for this task. We chose the a 16-channel 16-bit PWM shield from Adafruit. The 16-bits provide us much more range than the standard 8-bits of the Arduino. Additionally the Nano does not support as many channels so having 16 LEDs provides more flexibility in our measurements.
  	</p>
  	<h4>Constructing</h4>
  	<p>
  		We wanted to be able to illuminante a bacterial culture with a single led at multiple different intensities. We choose to grow our bacteria on microtiter plates. This choice was also affected by the fact that we had an access to a fluorometer (Thermo Scientific Varioskan<sup>5</sup>) that measures fluorescence out of such microtiter plates.
  	</p>
  	<p>
  		We designed our LED rig so that an unmodified 96 well microtiter plate with the test cultures can be easily inerted. Our LEDs are attached to a lid that can be placed on a standard microtiter plate. 16 LEDs will illuminate the wells on columns 11,9,7,5 and rows B, D, F, H.
  	</p>
  	<p>
  		Our LEDs' spectrum peak at the value of 470nm. This value is specified by the manufacturer but the spectra of individual leds can vary considerably. Therefore we will measure the spectra of our LED:s with a spectrophotometer to be sure of the excitation wavelength.
  	</p>
  	<p>
  		We have yet to invent a way to measure the LED intensity. If you have ideas, please contact us!
  	</p>
  	<ol>
  		<li> E.J. Olson, L.A. Hartsough, B.P. Landry, R. Shroff, J.J. Tabor, "Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals." Nature Methods 11(4), 449-455 (2014).</li>
		<li>Arduino Nano microcontroller, http://arduino.cc/en/Main/arduinoBoardNano</li>
		<li>Adafruit 16-channel PWM shield, https://learn.adafruit.com/adafruit-16-channel-pwm-slash-servo-shield?view=all</li>
		<li>S. Muthu, F. Schuurmans and M. Pashley, “Red, Green and Blue LED based white light generation: Issues and Control,” 37th Annual IEEEIAS meeting, Vol. 1, pp. 327 – 333 (2002)</li>
		<li>http://www.thermoscientific.com/content/tfs/en/product/varioskan-flash-multimode-reader.html</li>
  	</ol>
    <h3>Fluorescence measurements</h3>
    <p>
        Fluorescence measurements were carried out in Thermo Scientific Varioskan. It is a microtiter plate reader capable of exitation and measuring the corresponding fluorsecense at user defined wavelengths. All measurements were made with over night cell cultures diluted to the same cell density. In the LEDrig there are 16 different LEDs but also many additional wells that can be used for control samples. That way, we can ensure identical conditions between our test and control samples. We always used two controls: blue chromoprotein BioBrick (<a href="http://parts.igem.org/Part:BBa_K592009">BBa_K592009</a>) without a promoter as the negative control and constitutive GFP generator from Measurement Interlabs study (<a href="http://parts.igem.org/Part:BBa_I20260">BBa_I20260</a>) as the positive control. All experiments were carried out in <em>E. coli</em> TOP10 strain.
    </p>
    <p>All the Varioskan measurements used the following parameters:
        <ol>
            <li>Excitation wavelength: 470 nm</li>
            <li>Emission wavelength: 511 nm</li>
            <li>Bandwidth: 5nm</li>
        </ol>
    </p>
    <p>
        <strong>Minttu: voisitko kirjoittaa tähän, mikä GFP versio meillä on, millä aallonpituuksilla se eksitoidaan ja millä aallonpituudella se fluoresoi. Sitten kerro, millä perusteella me valittiin ne Varioskan-parametrit, jotka meillä on. Sä taisit olla paikalla silloin.</strong>
    </p>
</article>
<div class="update Methods"></div>
<div class="update Results"></div>
<article>
	<div class="link" id="Results"></div>

    <h2>Results</h2>
    <h3>GFP fluorescence in high OD<sub>600</sub> under different light intensities</h3>
    <p>
        In this experiment, we measured how cells will react to different intensities of blue light when grown in same conditions and very high cell density. In such conditions, the cells are not able to proliferate in a way that would affect to the measurements in short time period. This was the first experiment using different intensities of LED lights at 470 nm wavelength. Our purpose was to find out, how cells would respond to light in steady and high cell density and does the previous conditions affect to the fluorescence during a long time period. In addition, we were to observe the relative change between the highest and the lowest level of fluorescence. We had two kinds of samples: bacteria grown (overnight culture at 37C) in the darkness and in lit environment (usual lab lightning in addition to desk lamp directed to our sample). We also had a negative and a positive control for the fluorescence measurements.
    </p>
    <p>
        We constructed a testing device in order to estimate the concentration and the dynamics of the CI concentration. In the desting device, we replaced CI encoding sequence with GFP encoding sequence. The GFP was fused with LVA degradation tag. We excluded the lambda repressor and gene C parts. As a result, our testing device consisted of a blue light sensing element (constitutive promoter (<a href="http://parts.igem.org/Part:BBa_J23115">BBa_J23115</a>), RBS-YF1-RBS-FixJ (<a href="http://parts.igem.org/Part:BBa_K592016">BBa_K592016</a>) and a double terminator (<a href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</a>)) and the testing element (FixK2 promoter (<a href="http://parts.igem.org/Part:BBa_K592006">BBa_K592006</a>), RBS (<a href="http://parts.igem.org/Part:BBa_B0034">BBa_B0034</a>), GFP-LVA (<a href="http://parts.igem.org/Part:BBa_K082003">BBa_K082003</a>) and double terminator (<a href="http://parts.igem.org/Part:BBa_B0015">BBa_B0015</a>).
    </p>
    <p>
        At first, all the samples were diluted to OD<sub>600</sub> 2,4. They were inserted to different wells on microtiter plates, where control samples both were incubated in dark at two different wells each, and dark and light samples were incubated in 9 different intensities of blue 470 nm light. The first measurements were taken after 4 hours of incubation and a second mesurement was taken after 23 hours of incubation. Both incubations had the temerature at 37 C and shaking 200 rpm. <strong>**Light intensity values and their log10 values**</strong>
    </p>

    <div class="row">
        <div class="img-center">
            <img src="http://2014.igem.org/wiki/images/4/42/Aaltohelsinki_dark_4.png" class="img-responsive"></img>
            <p class="kuvateksti">
                Figure 1: GFP fluorescence of light and dark sample after 4 hours of incubation in different blue light intensities. Control samples were measured only in darkness. Positive control value (101,5) was excluded from the picture to preserve resolution.
            </p>
        </div>
    </div>

    <div class="row">
        <div class="img-center">
            <img src="http://2014.igem.org/wiki/images/4/49/Aaltohelsinki_1_23h.png" class="img-responsive"></img>
            <p class="kuvateksti">
                Figure 2: GFP fluorescence of light and dark sample after 23 hours of incubation in different blue light intensities. Control samples were measured only in darkness. Positive control value (176,9) was excluded from the picture to preserve resolution.
            </p>
        </div>
    </div>
    <p>
        From both figures we can make speculation that bacteria grown in darkness are more fluoreschent compared to the light cultures. This may be because of many different reasons. One possible explanation is that the bacteria have simply accumulated GFP during the preceding over night incubation. Therefore, all measurement points for dark cultures would be shifted to higher level of fluorescent. However, we also know that the GFP-LVA fusion half life is short <strong>**REFERENCE: How short?**</strong> compared to the time period. Thus, such higher GFP concentrations should fade over time and settle close to the actual production rate. These figures suggest that long term elevation of GFP production might take place in our system. We would need further experiments to verify this.
    </p>
    <p>
        When we observe the figure 1, we see a clear descending trend in the fluorescence towards higher intensities of light. This suggests that the bacteria indeed alter the rate of GFP production in response to the light even in very high cell densities. The brighter the light, the lower the GFP production and hence, the lower the fluorescence. The difference between bright light and darkness seems to be 1.5-2 fold. However, this effect could also be a result of the LED lights heating up the bacterial cultures and inhibiting proliferation and thus, leading to lower total GFP amount even if the per cell concentration would be constant. We also observe leaky production of GFP even in the bright light when we compare the fluorescence to the negative control strains without a GFP gene.
    </p>
    <p>
        <strong>**23 hour measurement. Quality compared to the previous one. Loss of the trend?</strong>
    </p>
    <p>
        <strong>Design for next experiment</strong>
    </p>

</article>
<div class="update Results"></div>
<div class="update Discussion"></div>
<article>
	<div class="link" id="Discussion"></div>

  <h2>Discussion</h2>
  <p>
        Mutaatiot operaattorialueisiin
        Plasmidissa ei DNA:ssa
        Solutason kohina solusyklissä

        Results: Possible room for improvement?
        LED spectrum, power measurements and intensity measurements?
        Heat production measurements?

        Red light
  </p>

</article>
<div class="update Discussion"></div>
<div class="update Conclusions"></div>
<article>
    <div class="link" id="Conclusions"></div>

  <h2>Conclusions</h2>

</article>
<div class="update Conclusions"></div>
<div class="update References"></div>
<article>
	<div class="link" id="References"></div>

  <h2>References</h2>

    <ul id="references">
        <li>
            Peking iGEM Team (2012), http://2012.igem.org/Team:Peking
        </li>
        <li>
            Uppsala-Sweden iGEM Team (2011), http://2011.igem.org/Team:Uppsala-Sweden
        </li>
        <li>
            SJTU BioX Shanghai iGEM Team, (2013), http://2013.igem.org/Team:SJTU-BioX-Shanghai
        </li>
        <li>
            Bell, C. E., Frescura, P., Hochschild, A., & Lewis, M. (2000). Crystal Structure of the λ Repressor C-Terminal Domain Provides a Model for Cooperative Operator Binding. Cell, 101(7), 801–811. doi:10.1016/S0092-8674(00)80891-0
        </li>
        <li>
            Dodd, I. B., Perkins, A. J., Tsemitsidis, D., & Egan, J. B. (2001). OcDodd, I. B., Perkins, A. J., Tsemitsidis, D., & Egan, J. B. (2001). <em>Octamerization of lambda CI repressor is needed for effective repression of P(RM) and efficient switching from lysogeny</em>. Genes & Development, 15(22), 3013–22. doi:10.1101/gad.937301tame. Genes & Development, 15(22), 3013–22. doi:10.1101/gad.937301
        </li>
        <li>
            Hirose, Y., Shimada, T., Narikawa, R., Katayama, M., & Ikeuchi, M. (2008). <em>Cyanobacteriochrome CcaS is the green light receptor that induces the expression of phycobilisome linker protein</em>. Proceedings of the National Academy of Sciences of the United States of America, 105(28), 9528–33. doi:10.1073/pnas.0801826105
        </li>
        <li>
            Johnson, A. D., Meyer, B. J., & Ptashne, M. (1979). <em>Interactions between DNA-bound repressors govern regulation by the lambda phage repressor</em>. Proceedings of the National Academy of Sciences of the United States of America, 76(10), 5061–5. Retrieved from http://www.pubmedcentral.nih.gov/articlerender.fcgi?artid=413079&tool=pmcentrez&rendertype=abstract
        </li>
        <li>
            Johnson, A. D., Poteete, A. R., Lauer, G., Sauer, R. T., Ackers, G. K., & Ptashne, M. (1981). <em>λ Repressor and cro—components of an efficient molecular switch</em>. Nature, 294(5838), 217–223. doi:10.1038/294217a0
        </li>
        <li>
            Keiler, K. C., Waller, P. R., & Sauer, R. T. (1996). Role of a peptide tagging system in degradation of proteins synthesized from damaged messenger RNA. Science (New York, N.Y.), 271(5251), 990–3. Retrieved from http://www.ncbi.nlm.nih.gov/pubmed/8584937
        </li>
        <li>
            Levskaya, A., Chevalier, A. A., Tabor, J. J., Simpson, Z. B., Lavery, L. A., Levy, M., … Voigt, C. A. (2005). <em>Synthetic biology: engineering Escherichia coli to see light</em>. Nature, 438(7067), 441–2. doi:10.1038/nature04405
        </li>
        <li>
            Möglich, A., Ayers, R. A., & Moffat, K. (2009). <em>Design and signaling mechanism of light-regulated histidine kinases</em>. Journal of Molecular Biology, 385(5), 1433–44. doi:10.1016/j.jmb.2008.12.017
        </li>
        <li>
            Olson, E. J., Hartsough, L. A., Landry, B. P., Shroff, R., & Tabor, J. J. (2014). Characterizing bacterial gene circuit dynamics with optically programmed gene expression signals. Nature Methods, 11(4), 449–55. doi:10.1038/nmeth.2884
        </li>
        <li>
            Parsell, D. A., Silber, K. R., & Sauer, R. T. (1990). Carboxy-terminal determinants of intracellular protein degradation. Genes & Development, 4(2), 277–286. doi:10.1101/gad.4.2.277
        </li>
        <li>
            Tabor, J. J., Levskaya, A., & Voigt, C. A. (2011). <em>Multichromatic control of gene expression in Escherichia coli</em>. Journal of Molecular Biology, 405(2), 315–24. doi:10.1016/j.jmb.2010.10.038
        </li>
        <li>
            Zopf, C. J., Quinn, K., Zeidman, J., & Maheshri, N. (2013). <em>Cell-cycle dependence of transcription dominates noise in gene expression</em>. PLoS Computational Biology, 9(7), e1003161. doi:10.1371/journal.pcbi.1003161
        </li>
        **Linkit proteiineista, promoottoreista ja BioBrickeistä vastaaville sivuille maininnanyhteydessä?**
    </ul>

</article>
<div class="update References"></div>
